_rules/80f0bf.md

```diff
@@ -0,0 +1,112 @@
+---
+name: No auto-play audio
+test_type: composite
+
+description: |  
+  This rule checks that auto-play audio does not last for more than 3 seconds, or it has a control mechanism.
+
+success_criterion: 
```

Comment:
change to `accessibility_requirements`

Reply:
accessibility_requirements:
  wcag20:1.4.2: # Audio Control
    forConformance: true
    failed: not satisfied
    passed: further testing needed
    inapplicable: further testing needed